pages/dkp/kommander/1.3/projects/platform-services/platform-services-catalog/kudo-kafka/index.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: layout.pug
+navigationTitle: KUDO Kafka
+render: mustache
+title: KUDO Kafka
+menuWeight: 3
+excerpt: Day 2 Operations of KUDO Kafka
+beta: false
+---
+
+<!-- markdownlint-disable MD018 -->
+## KUDO Kafka
+
+Kommander Catalog adds integration for [KUDO Kafka Operator](https://github.com/mesosphere/kudo-kafka-operator/), which simplifies day 2 operations of [Apache Kafka](https://kafka.apache.org/). 
+
+#include /dkp/kommander/1.3/include/kudo-intro.tmpl
+
+It is **strongly recommended** to view the [KUDO Kafka Documentation](https://kudo.dev/docs/runbooks/kafka/install.html) which covers the KUDO Kafka Operator in-depth. This document covers the integration aspects of KUDO Kafka Operator with D2iQ Kommander.
+
+### Kommander Catalog
+
+KUDO Kafka is located in the Kommander Catalog.   To access the catalog: 
```

Comment:
```suggestion
KUDO Kafka is located in the Kommander Catalog. To access the catalog: 
```

Reply:
Not to bring up a holy war here but this is what we were taught in Grade 9 (I think) typesetting class. ;)

https://www.thesaurus.com/e/writing/is-there-1-space-or-2-after-a-period/